extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -203,11 +196,11 @@ export class AmpGeo extends AMP.BaseElement {
       const overrideGeoMatch = GEO_HOTPATCH_STR_REGEX.exec(
         getMode(this.win).geoOverride.toLowerCase()
       );
-      if (overrideGeoMatch && overrideGeoMatch[1]) {
-        this.country_ = overrideGeoMatch[1];
-        if (overrideGeoMatch[3]) {
+      if (overrideGeoMatch[1]) {
+        this.country_ = overrideGeoMatch[1].toLowerCase();
```

Comment:
The hot patched value should never be upperCase. But having another layer of protection sounds good.